common/config/config.go

```diff
@@ -107,6 +112,26 @@ func (c *Config) GetDiscoveryURL() string {
 	return fmt.Sprintf("%s://%s:%d/ngsi9", protocol, hostip, port)
 }
 
+func (c *Config) GetDesignerURL() string {
+        protocol := "http"
+        port := c.Designer.WebSrvPort
+        if c.HTTPS.Enabled == true {
+                protocol = "https"
+                port = c.Designer.HTTPSPort
+        }
+
+        hostip := c.InternalIP
+        if c.CoreSerivceIP != "" {
+                hostip = c.CoreSerivceIP
+        }
+
+        if c.Discovery.HostIP != "" {
+                hostip = c.Designer.HostIP
```

Comment:
looking logic is wrong. please check it.

Reply:
Comment Incorporated